packages/twenty-server/src/core/user/user.entity.ts

```diff
@@ -87,4 +92,12 @@ export class User {
 
   @Field(() => WorkspaceMember, { nullable: true })
   workspaceMember: WorkspaceMember;
+
+  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.user, {
+    onDelete: 'CASCADE',
+  })
```

Comment:
we don't want to delete user record when deleting related userWorkspaces record
```suggestion
  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.user)
```